Formula/mingw-w64-binutils.rb

```diff
@@ -11,18 +11,40 @@ class MingwW64Binutils < Formula
     sha256 "39e04cd72f16ce87abac183cfa9d1bfe3b979378a938f4a8b987b79ea696261d" => :yosemite
   end
 
+  option "without-i686", "Compile without i686 mingw target"
+  option "without-x86_64", "Compile without x86_64 mingw target"
+  option "without-multilib", "Compile x86_64 mingw target without multilib"
+
+  def targets
+    if build.without?("i686") && build.without?("x86_64")
+      odie "Options --without-i686 and --without-x86_64 are mutually exclusive"
+    end
+    archs = []
+    archs.push("i686-w64-mingw32") if build.with?("i686")
```

Comment:
We usually use `archs << "i686-w64-mingw32'` etc.; it'd be nice to maintain the convention.